presto-docs/src/main/sphinx/release/release-0.236.rst

```diff
@@ -0,0 +1,26 @@
+=============
+Release 0.236
+=============
+
+General Changes
+_______________
+* Fix query failure due to searched case expression with large number of cases.
```

Comment:
This is still too complicated for me to understand. Maybe I am dumb - @tdcmeehan, @rschlussel - your thoughts on this one?

Reply:
@mayankgarg1990 "searched case expression with large number of cases" is basically:
```
CASE
  WHEN ... THEN ...
  WHEN ... THEN ...
  ...
END
```
with lots of WHEN clauses.

Reply:
the confusing part of this for me is "searched case expression".  Seems it just means "case expression", so I would get rid of the word "searched"

```
Fix query failure for case expression with large number of cases. 
```